### PR TITLE
Sidebar component 

### PR DESCRIPTION
**Pull request by jarhun88**
### What problem are you solving in this PR?
Create Sidebar component 

### Related issues/PRs:
Resolves https://github.com/siddhero97/CS436I-Memofy/issues/9

### How is this accomplished and what led to this approach?
1. Created Sidebar component (6319d96b1bdbe9c422a78f7819553132ffc86b28)

### What are the risks in this PR?
1. Too ugly

### Before-I-merge checklist
- [x] I have [self-tophatted]

### CODE REVIEW

**Comment by jarhun88** [comment]
the sidebar isnt responsive, not sure how to do this with css-grid, any tips?

**Comment by siddhero97** [comment]
https://www.youtube.com/watch?v=alsxCvrbeko

**Comment by siddhero97** [comment]
If we want to just do the mvp for now. This should be Gucci. But if we want to make it pretty. I think we should use Bootstrap. It will make life much more easier.

File: client/src/sections/Fridge/Sidebar/Sidebar.tsx [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
 export default function Sidebar() {
+
+  const [categories, setCategories] = useState([
+    "Meats", "Fruits", "Veggies"
+  ])
+
   return (
       <div className='sidebar'>
-            Sidebar placeholder
+        {categories.map( i => (
```
Comment (KwanLam17) [inline client/src/sections/Fridge/Sidebar/Sidebar.tsx:14]: Lets move the logic out of the render and make a constant on top for the markup.

```
const categoriesMarkup = categories.map((category, i) => {
    return (
      <div className="category" key={i}>{category}</div>
    );
  })

return (
    <div className='sidebar'>
       {categoriesMarkup}
       <div className="category" onClick={() => setCategories(categories => [...categories, "apple"])}>+</div>
    </div>
);
```

**Pull request merged**